package.json

```diff
@@ -62,17 +59,24 @@
     "sass-loader": "~3.1.2",
     "style-loader": "~0.12.3",
     "svg-prep": "~1.0.0",
-    "transform-jest-deps": "^2.1.0",
     "webpack": "~1.12.0"
   },
+  "devDependencies": {
+    "history": "~1.9.1",
+    "http-server": "~0.8.5",
+    "jest-cli": "^0.7.1",
+    "transform-jest-deps": "^2.1.0"
+   },
   "scripts": {
     "dev": "node ./Parse-Dashboard/index.js & webpack --config webpack/build.config.js --devtool eval-source-map --progress --watch",
     "dashboard": "node ./Parse-Dashboard/index.js & webpack --config webpack/build.config.js --progress --watch",
+    "dist": "npm run build && cp -r production/bundles Parse-Dashboard/public/bundles",
     "pig": "http-server ./PIG -p 4041 -s & webpack --config webpack/PIG.config.js --progress --watch",
     "build": "NODE_ENV=production webpack --config webpack/production.config.js && webpack --config webpack/PIG.config.js",
     "test": "NODE_PATH=./node_modules jest",
     "generate": "node scripts/generate.js",
     "prepublish": "webpack --config webpack/publish.config.js --progress",
+    "postinstall": "if [ -n \"$POST_INSTALL_BUILD\" ]; then npm run dist; fi",
```

Comment:
you could use: "heroku-postbuild" (as described [here](https://devcenter.heroku.com/articles/nodejs-support#customizing-the-build-process))